setup.py

```diff
@@ -56,8 +56,9 @@
         'thriftrw>=0.4,<2',
         'threadloop>=1,<2',
 
-        # tracing deps
-        'opentracing>=1.0rc4,<2'
+        # tracing deps - pinned due to upcoming API changes
+        'opentracing>=1.0rc4,<1.1',
+        'opentracing_instrumentation>=1.1,<1.2',
```

Comment:
I'm still not sure why you're pinning to the minor here - are you guys planning breaking changes within a `1.x`?